### PR TITLE
Bracket

### PR DESCRIPTION
**Pull request by tristan-gill**
Added `playoffSeries` template and pages. I didn't want to mess around with the existing pages too much and there is some code difference between a normal series and a playoff series. There is a small amount of duplication in the templates+pages between series and playoff series. I think it's a low enough amount, it makes some sense that a series and playoff are shown differently, plus they have different routes. I also didn't want to re-write everything :X 

Added a new page called bracket. This displays all of the `series` that exist for a defined season. It uses round and match_number to determine how to place the matches/series in the bracket. 
 - Round determines which column the match goes into
 - Match number determines the order of the matches within that column. 
Using incorrect numbers will moderately disfigure the layout of the bracket, but if done correctly it should look something like this.

In order for the bracket to resemble the challonge bracket someone has to make all the series. All admins have the ability to do so. Admins also see an edit tag on the bracket.

image

[OLD PR can be found here](https://github.com/redditdota2league/server/pull/149)

### CODE REVIEW

**Comment by tristan-gill** [comment]
@kabobrocks @dambrisco Not sure why I felt the necessity to remake this PR, but alas. 
I left in the pages and templates for the playoff_series. I didnt want to re-write all of my PR or mess with the existing pages too much. Plus they have different routes.
Lemme know what you think. It would be nice to get this out in time for this playoff as well

**Comment by kabobrocks** [comment]
Discussed over slack updating the footer and also adding a link to bracket on the division hub

File: src/pages/playoffSeries.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+  const roundOne = series.filter((matchup) => {
+    return matchup.round === 1;
+  });
+
+  const numberOfMatchups = roundOne.length * 2;
+  const numRounds = Math.ceil(Math.log2(numberOfMatchups));
+
+  let remainingSeries = series.slice(roundOne.length - 1);
+  let currentRoundNum = 2;
```
Comment (dambrisco) [inline src/pages/playoffSeries.js]: Is this supposed to be hard-coded?

Comment (tristan-gill) [reply src/pages/playoffSeries.js]: Yea, we start processing the rounds at the second.
I'm doing some tweaks that kabo mentioned above, I'll add some more documentation about this processing at the same time

**Comment by tristan-gill** [comment]
@kabobrocks @dambrisco Addressed your comments.

**Pull request merged**